vnext/Microsoft.ReactNative/RedBox.cpp

```diff
@@ -60,31 +60,32 @@ struct RedBox : public std::enable_shared_from_this<RedBox> {
     m_popup = xaml::Controls::Primitives::Popup{};
 
     const winrt::hstring xamlString =
-        L"<Grid Background='{ThemeResource ContentDialogBackground}'"
-        L"  xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation'"
-        L"  xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'"
-        L"  RequestedTheme='Dark'>"
-        L"  <Grid.ColumnDefinitions>"
-        L"    <ColumnDefinition Width='*'/>"
-        L"    <ColumnDefinition Width='*'/>"
-        L"  </Grid.ColumnDefinitions>"
-        L"  <Grid.RowDefinitions>"
-        L"    <RowDefinition Height='*'/>"
-        L"    <RowDefinition Height='Auto'/>"
-        L"  </Grid.RowDefinitions>"
-        L"  <ScrollViewer Grid.Row='0' Grid.ColumnSpan='2' HorizontalAlignment='Stretch'>"
-        L"    <StackPanel HorizontalAlignment='Stretch'>"
-        L"      <StackPanel Background='#EECC0000' HorizontalAlignment='Stretch' Padding='15,35,15,15' x:Name='StackPanelUpper'>"
-        L"        <TextBlock x:Name='ErrorMessageText' FontSize='14' Foreground='White' IsTextSelectionEnabled='true' TextWrapping='Wrap'/>"
-        L"        <Border Padding='15,15,15,0'/>"
-        L"        <TextBlock x:Name='ErrorStackText' FontSize='12' FontFamily='Consolas' Foreground='White' IsTextSelectionEnabled='true' TextWrapping='Wrap'/>"
-        L"      </StackPanel>"
-        L"      <StackPanel HorizontalAlignment='Stretch' x:Name='StackPanel' Margin='15' />"
-        L"    </StackPanel>"
-        L"  </ScrollViewer>"
-        L"  <Button x:Name='DismissButton' Grid.Row='1' Grid.Column='0' HorizontalAlignment='Stretch' Margin='15' Style='{StaticResource ButtonRevealStyle}'>Dismiss</Button>"
-        L"  <Button x:Name='ReloadButton' Grid.Row='1' Grid.Column='2' HorizontalAlignment='Stretch' Margin='15' Style='{StaticResource ButtonRevealStyle}'>Reload (NYI)</Button>"
-        L"</Grid>";
+        LR"(
+        <Grid Background='#FF333333'
+          xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation'
+          xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'
+          RequestedTheme='Dark'>
+          <Grid.ColumnDefinitions>
+            <ColumnDefinition Width='*'/>
+            <ColumnDefinition Width='*'/>
+          </Grid.ColumnDefinitions>
+          <Grid.RowDefinitions>
+            <RowDefinition Height='*'/>
+            <RowDefinition Height='Auto'/>
+          </Grid.RowDefinitions>
+          <ScrollViewer Grid.Row='0' Grid.ColumnSpan='2' HorizontalAlignment='Stretch'>
+            <StackPanel HorizontalAlignment='Stretch'>
+              <StackPanel Background='#FFF35369' HorizontalAlignment='Stretch' Padding='15,35,15,15' x:Name='StackPanelUpper'>
```

Comment:
is this right? this color is a mustard yellow, but the redbox is supposed to be for errors...and it has red in the name :)

Reply:
The new Logbox replaced Redbox and Yellowbox. I think we still have a native redbox for errors that occur when loading JS that can't render the Logbox. Is that what this is? If so, I wouldn't expect a yellow color in native code (although my understanding of Logbox / redbox is surely incomplete).

Reply:
I want to make sure that errors coming from metro etc, are shown distinctly as errors. Here's what we have today for one type of error from metro.
![image](https://user-images.githubusercontent.com/22989529/84457975-c9059400-ac18-11ea-98f9-5b4ec0c4dc7f.png)

Reply:
Its not a yellow, its a pale red.  I was updating the native redbox colors to match the ones used by logbox errors.  LogBox errors use a lighter red.  But taking another look at core it looks like core native redbox doesn't use the same color as logbox errors.  But they did update the color of the native redbox.  -- So I'll change to use that color. (#D01926)

https://github.com/facebook/react-native/blob/40b36a040f06aae03be1a09f14e48eb0c652de96/ReactAndroid/src/main/res/devsupport/layout/redbox_item_title.xml#L10